### ISSUE TITLE
Main transmission packets

### ISSUE THREAD

**Issue by wiresboy** [HuskieRobotics/roborioExpansion#48]
First, this isn't even an accurate name any more. It is by no means the "main" packet. And the naming is not consistent across our different code bases. (Spec labels it as `Main transmission`, SPIN refers to it as `give_data_func`, etc.)

Second, what exactly do we want to transmit with this? It seems to me that all of its functionality has been superseded elsewhere. **I vote for deprecating this packet.** 

Thoughts?

**Comment by gcschmit** [comment]
I agree; deprecate it. It appears to be specific to our team's use of the board in the past.

**Comment by gcschmit** [comment]
Just remove it.
